code/drasil-example/Drasil/DblPendulum/Concepts.hs

```diff
@@ -9,10 +9,27 @@ pendulum :: CI
 pendulum = commonIdeaWithDict "pendulum" (pn "Pendulum") "Pendulum" [physics]
 
 concepts :: [IdeaDict]
-concepts = [nw rod] 
+concepts = map nw [rod, horizontal, vertical] ++ map nw defs 
        
-rod :: NamedChunk
+rod, horizontal, vertical :: NamedChunk
 rod = nc "rod" (cn' "rod")
+horizontal = nc "horizontal" (cn "horizontal") 
+vertical = nc "vertical" (cn "vertical") 
+---
+
+defs:: [ConceptChunk]
+defs = [shm, amplitude, component, arcLen, motion]
+
+shm, amplitude, component, arcLen, motion :: ConceptChunk
+shm = dcc "SHM" (nounPhraseSP "simple harmonic motion") ("repetitive movement back and forth through" ++
+                                                         "an equilibrium position such that the maximum" ++
+                                                         "displacement on one side is equal to the maximum" ++
+                                                         "displacement on the other side")
+amplitude = dcc "amplitude" (nounPhraseSP "amplitude") ("the maximum displacement of a body measured from" ++
+                                                        "its equilibrium position")
+component = dcc "component" (nounPhrase "component" "components") "one part or direction of a vector"
```

Comment:
I don't like this definition of vector components, since a component isn't a direction, but the projection of the vector onto one of the basis vectors of the coordinate system.  How about something like:

The scalar quantity defining the contribution of the vector in one of the coordinate directions

I actually found this one a bit tricky to say in a single sentence.

Reply:
I think the definition of vector component belongs to `Math`.